workflows/laava/workflow.json

```diff
@@ -192,12 +192,12 @@
     },
     "itr_label_2": {
       "title": "ITR region label 2",
-      "description": "Right ITR label in annotation BED",
-      "help_text": "Region label used in the vector annotation BED file to indicate the second (i.e. right, 3') inverted terminal repeat (ITR). This label is often 'ITR' or 'ITR-R', but can vary. Case-sensitive.",
+      "description": "Second ITR label in annotation BED",
+      "help_text": "Region label used in the vector annotation BED file to indicate the second (i.e. right, 3') inverted terminal repeat (ITR). This label is often 'ITR' or 'ITR-R', but can vary. Case-sensitive. If the same label is used for both ITRs in the annotation BED (e.g. 'ITR'), it's not necessary to specify it again here; ITR region label 1 will correctly match both ITRs in that case.",
       "format": "text",
       "hidden": false,
       "required": false,
-      "default": "ITR",
```

Comment:
We should keep default to ITR for both first and second ITR.
What is the rational to have first and second, rather than previously left and right?

Reply:
From looking at a circular diagram in SnapGene, users might confuse or swap the order of the ITRs in the exported BED/Genbank file. This way, "ITR-R" followed by "ITR-L" will work, even if swapped. As a side effect, entering "ITR" for both itr_label_1 and itr_label_2 works the same as leaving itr_label_2 blank.